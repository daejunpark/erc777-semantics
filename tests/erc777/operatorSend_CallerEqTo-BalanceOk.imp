test.preamble();

// Testing operatorSend(From,To,Value,"","") when To = caller.
// All conditions are met, so it is expected to pass.

test.makeCaller(9);           // switch to 9
test.operatorSend(7, 9, 23, "", "");  // send 23 from 7 to 9
