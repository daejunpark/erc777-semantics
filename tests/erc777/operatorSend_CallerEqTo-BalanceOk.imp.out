Changing the caller to 9
9 is sending 23 from 7 to 9 (data:""),  (operatorData:"")
Balance of 7 is 47
Balance of 9 is 113
